website/views/organization.py

```
@@ -3930,3 +3955,189 @@ def extract_payment_info_from_comments(self, issue, owner, repo_name, issue_numb
+        app_desc = app.get("description", "")
+        commands = []
+
+        try:
+            cmd_resp = slack_safe_call(slack, "apps.commands.list", params={"app_id": app_id})
+            commands = cmd_resp.get("commands", [])
+        except SlackApiError as e:
+            logger.warning("Slack command fetch failed for %s: %s", app_id, e)
+        except requests.exceptions.RequestException:
+            logger.warning("Network error while fetching commands for %s", app_id)
+            continue
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)